packages/react/toast/src/Toast.tsx

```diff
@@ -195,34 +204,63 @@ const ToastViewport = React.forwardRef<ToastViewportElement, ToastViewportProps>
       }
     }, [context.isClosePausedRef]);
 
+    const getSortedCandidates = React.useCallback(
+      ({ direction }: { direction: 'forwards' | 'backwards' }) => {
+        const items = getItems();
+        const candidateGroups = items.map((item) => {
+          const itemNode = item.ref.current!;
+          const itemGroup = [itemNode, ...getTabbableCandidates(itemNode)];
+          return direction === 'forwards' ? itemGroup.reverse() : itemGroup;
+        });
+        return (direction === 'forwards' ? candidateGroups : candidateGroups.reverse()).flat();
```

Comment:
You mean because it depends if you're thinking about the user's tabbing direction, vs the direction focus is actually moving through elements? forwards/backwards in time in terms of toast creation?

Perhaps, if we qualify it a bit more like `tabbingDirection` that would make it clearer?